pkg/tests/base/cases/configcontext/upstream/test-chart-2.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: kots.io/v1beta1
+kind: HelmChart
+metadata:
+  name: test-chart-2
+spec:
+  # exclude: 'repl{{ ConfigOptionEquals "helm" "no" }}'
+  # chart identifies a matching chart from a .tgz
+  chart:
+    name: test-chart-2
+    chartVersion: 0.1.1
+  useHelmInstall: false
```

Comment:
I may be off.. you have a chart with helmhooks but useHelmInstall set to false and added a different namespace ? that is when this regression happens?

Reply:
Removed Helm annotations.